### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by frewsxcv**
Cleanup and fix for #11

### CODE REVIEW

File: ticketing/fields.py [hunk 1]
```diff
@@ -39,14 +39,15 @@ def pre_save(self, model_instance, add):
...
 
-    def db_type(self, *args, **kwargs):
-        connection = kwargs.get('connection', None)
-        if connection and connection.settings_dict['ENGINE'] == 'django.db.backends.mysql':
+    def db_type(self, connection, *args, **kwargs):
+        if _conn_is_mysql(connection):
             return 'bigint UNSIGNED'
         else:
-            return models.BigIntegerField.db_type(self, *args, **kwargs)
+            return models.BigIntegerField.db_type(
```
Comment (streeter) [inline ticketing/fields.py]: Should this also use `super(TicketField, self).db_type`?

**Comment by frewsxcv** [comment]
How does this look? I'm going to manually test it now. It shouldn't have taken this long, but I blame it on my inexperience creating custom model fields.

File: ticketing/fields.py [hunk 2]
```diff
@@ -20,33 +24,30 @@ def __init__(self, sequence=None, **kwargs):
...
+        self.sequence = sequence or conf.DEFAULT_SEQUENCE
         kwargs['editable'] = False
-        models.BigIntegerField.__init__(self, **kwargs)
+        super(TicketField, self).__init__(**kwargs)
 
     def pre_save(self, model_instance, add):
         """
         When saving the DB, if we are adding this instance, get a ticket
         value for this field. Otherwise pull it from the model instance.
+        If the field on the instance already has a valude, we won't
```
Comment (streeter) [inline ticketing/fields.py]: Typo here.

**Comment by streeter** [comment]
LGTM

**Pull request merged**